docs/getting_started.rst

```diff
@@ -75,3 +75,45 @@ Subscribe to tag changes
 .. literalinclude:: ../examples/tag/subscribe_to_tag_changes.py
    :language: python
    :linenos:
+
+Data Frame API
+-------
+
+Overview
+~~~~~~~~
+
+The :class:`.DataFrameClient` class is the primary entry point of the Data Frame API.
+
+When constructing a :class:`.DataFrameClient`, you can pass an
+:class:`.HttpConfiguration` (like one retrieved from the
+:class:`.HttpConfigurationManager`), or let :class:`.DataFrameClient` use the
+default connection. The default connection depends on your environment.
+
+With a :class:`.DataFrameClient` object, you can:
+
+* Retrieve information about available API operations
+
+* Query, create, modify, and delete Data Frame Tables and associated metadata
+
+* Query, query decimated, and append Data Frame Table data
```

Comment:
Since we have two (or three) kinds of queries, maybe we can distinguish between them a little more. What do you think about something like this?

```suggestion
* Create and delete Data Frame Tables.

* Modify table metadata and query for tables by their metadata.

* Append rows of data to a table, query for rows of data from a table, and decimate table data.
```

It could also be useful to get Melissa's feedback for this one.

Reply:
It won't let me add Melissa to the review for some reason, but I'll leave this one open until I get her sign-off.